tests/package.json

```diff
@@ -9,7 +9,7 @@
     "dist/**/*.d.ts"
   ],
   "scripts": {
-    "install": "tsc --build tsconfig.json"
+    "postinstall": "tsc --build tsconfig.json"
```

Comment:
Looks like this affects [installation steps](https://github.com/eclipse/che-che4z-lsp-for-cobol/blob/development/tests/README.md#2-install-all-dependencies).

Reply:
Yeah, I'll add in further PR all the info :)